### PR TITLE
fix cocos2d-x and cocosdenshion ios project setting, to avoid symbol not...

### PR DESCRIPTION
**Pull request by stubma**
fix cocos2d-x and cocosdenshion ios project setting, to avoid symbol not found error

### CODE REVIEW

File: CocosDenshion/proj.ios/CocosDenshion.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -212,7 +212,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
-				ARCHS = "$(ARCHS_STANDARD_32_BIT)";
```
Comment (dumganhar) [inline CocosDenshion/proj.ios/CocosDenshion.xcodeproj/project.pbxproj:215]: Why could deleting this line resolve the issue?
When will the issue appear?

**Comment by stubma** [comment]
test environment: xcode 5.0.1, target device is iphone 5
create a cocos2d-x project, drag cocos2d-x and cocosdenshion project into it, try to run it on device

get link error:


cocos2d-x and cocosdenshion ios project has a non-default architecture setting, I think it is not necessary. delete them will return it to default value, works fine.

**Comment by dumganhar** [comment]
Hi @stubma 
I tested project created by `project_creator` of master branch. And following your step, I could not reproduce this issue. Any comments?

**Comment by stubma** [comment]
don't forget call some cocosdenshion api in test code

**Comment by dumganhar** [comment]
Yep, I invoking `SimpleAudioEngine::end();` method in AppDelegate.cpp.

**Comment by stubma** [comment]
project_creator includes cocosdenshion source directly. I don't use it that way. Try include cocosdenshion project.

**Comment by dumganhar** [comment]
Yep, I tried it same as you. But I could not get the linking error results.

**Comment by stubma** [comment]
because the project generated by project-creator has a different architecture setting. 
image

If you restore test project architecture setting to default value, error should be seen

**Pull request closed without merging**